### PR TITLE
222 fix

### PR DESCRIPTION
**Pull request by KrisGraySFDC**

### CODE REVIEW

File: src/LightningInspectorInjectedScript.js [hunk 1]
```diff
@@ -1072,7 +1072,9 @@ function addActionRules() {
         var removeClassName = 'auraDevToolServiceHighlight3';
         var addClassName = 'auraDevToolServiceHighlight4';
         var element = event.target;
-        element.classList.remove(removeClassName);
-        element.classList.remove(addClassName);
+        if (element) {
+            element.classList.remove(removeClassName);
```
Comment (mysticflute) [inline src/LightningInspectorInjectedScript.js]: names are a bit weird for them both to be removed

File: src/aura/gatherer/ComponentSerializer.js [hunk 2]
```diff
@@ -209,7 +206,7 @@ export default class ComponentSerializer {
                 // BODY
                 else if (configuration.body) {
                     const body = getBody(component);
-                    output.attrbiutes['body'] = body.attributes;
+                    output.attributes['body'] = body.attributes;
```
Comment (mysticflute) [inline src/aura/gatherer/ComponentSerializer.js]: lol

File: src/aura/gatherer/ComponentSerializer.js [hunk 3]
```diff
@@ -384,3 +378,21 @@ function getBody(component) {
...
     };
 }
+
+function getAttributeExpression(component, attribute) {
+    let expression = null;
+    try {
+        expression = $A.componentService.getAttributeExpression(component, attribute);
+    } catch (e) {
+        // getAttributeExpression() in 222 does not support interop component
+        // lucily the attributes property is not minified, giving us
```
Comment (mysticflute) [inline src/aura/gatherer/ComponentSerializer.js]: lucy

File: src/aura/gatherer/ComponentSerializer.js [hunk 4]
```diff
@@ -384,3 +378,21 @@ function getBody(component) {
...
+        // the ability to monkey patch it.
+        // In 224 getAttributeExpression() should support interop and we can remove this.
+        if (component && component.attributes) {
+            if ($A.util.isExpression(component.attributes[attribute])) {
```
Comment (mysticflute) [inline src/aura/gatherer/ComponentSerializer.js]: assuming $A.util.isExpression can handle undefined, if the attribute is not present

File: src/LightningInspectorInjectedScript.js (hunk 1 above)
Comment (KrisGraySFDC) [reply src/LightningInspectorInjectedScript.js]: They show the highlights when you hover over them in the component tree.

If you don't remove both, then the page becomes unusable.

File: src/aura/gatherer/ComponentSerializer.js (hunk 2 above)
Comment (KrisGraySFDC) [reply src/aura/gatherer/ComponentSerializer.js]: I blame JY, but it was probably me. :)

**Pull request merged**